### PR TITLE
Fixes varediting atoms to null locations not working

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -54,9 +54,12 @@
 				return TRUE
 			return FALSE
 		if("loc")
-			if(var_value == null || istype(var_value, /atom))
+			if(istype(var_value, /atom))
 				forceMove(var_value)
 				return TRUE
+			else if(var_value == null)
```
Comment (ACCount12) [inline code/game/atoms_movable.dm]: Maybe `isnull` would be better there?

**Comment by optimumtact** [comment]
you shouldn't be moving objects to nullspace, literally nothing will handle it appropriately and it will cause runtimes

**Comment by ACCount12** [comment]
It's badmin shenanigans. There are way too many ways to break shit with VV, enough of them to make nullspace trips look harmless.

**Comment by Shadowlight213** [comment]
Not really a fix.

**Comment by Cyberboss** [comment]
This is dumb, just delete it properly

**Comment by Shadowlight213** [comment]
@Cyberboss the only situation i can think of for this to bu useful is testing the secret room

**Pull request closed without merging**